tests/integration/cases/example_maven_app_sbom_tutorial/dependencies.json

```diff
@@ -1,17 +1,17 @@
 [
     {
-        "id": "com.google.guava:guava",
-        "purl": "pkg:maven/com.google.guava/guava@32.1.2-jre?type=jar",
-        "path": "https://github.com/google/guava",
+        "id": "io.github.behnazh-w.demo:jackson-databind",
+        "purl": "pkg:maven/io.github.behnazh-w.demo/jackson-databind@1.0?type=jar",
+        "path": "https://github.com/behnazh-w/maven-demo",
         "branch": "",
         "digest": "",
         "note": "",
         "available": "AVAILABLE"
     },
     {
-        "id": "io.github.behnazh-w.demo:jackson-databind",
-        "purl": "pkg:maven/io.github.behnazh-w.demo/jackson-databind@1.0?type=jar",
-        "path": "https://github.com/behnazh-w/maven-demo",
+        "id": "org.apache.logging.log4j:log4j-core",
```

Comment:
I wonder if it's expected for the guava dependency to be replaced with logging-log4j2?